inc/class_session.php

```diff
@@ -65,7 +65,14 @@ function init()
 		// Attempt to find a session id in the cookies.
 		if(isset($mybb->cookies['sid']) && !defined('IN_UPGRADE'))
 		{
-			$sid = $db->escape_string($mybb->cookies['sid']);
+			$cookies = explode('; ', $_SERVER['HTTP_COOKIE']);
```

Comment:
This looks like a weird workaround to me. If there is an issue with `$mybb->cookies` (likely affecting other features) that should be fixed instead.